static/app/components/replays/useReplaysCount.tsx

```diff
@@ -1,12 +1,18 @@
 import {useCallback, useEffect, useMemo, useState} from 'react';
 import * as Sentry from '@sentry/react';
 
-import {IssueCategory, Organization} from 'sentry/types';
+import {DateString, IssueCategory, Organization} from 'sentry/types';
 import toArray from 'sentry/utils/toArray';
 import useApi from 'sentry/utils/useApi';
 
 type Options = {
   organization: Organization;
+  datetime?: {
+    end: DateString;
+    start: DateString;
+    statsPeriod: undefined;
+  };
+  extraConditions?: string;
```

Comment:
cc @cmanallen will datetime ranges, and general `extraConditions` hurt our query optimizations?

Reply:
It depends.  The fewer the projects and the tighter the date range the better we perform.  But, there's an exception.  Replay-ids are indexed so they have the potential to return more quickly.  It depends on the number of replays and how many segments they have.

Let's just deploy it and find out.